tests/integration/commands/deploy/deploy.test.ts

```diff
@@ -953,6 +954,71 @@ describe.skipIf(process.env.NETLIFY_TEST_DISABLE_LIVE === 'true').concurrent('co
     })
   })
 
+  test('should upload blobs when saved into .netlify directory and using --auth flag', async (t) => {
```

Comment:
this test alone was failing (at least in GHA) without code changes done in 2nd commit